tests/test_tools.sh

```diff
@@ -206,8 +208,9 @@ create_template_loader_json
 run_command_and_check_results "vcf2genomicsdb_init -w $WORKSPACE -S $SAMPLE_DIR -o -t $TEMPLATE" 2 85 24 85 "#17"
 assert_true $(grep '"segment_size": 400' $WORKSPACE/loader.json | wc -l) 1 "Test #16 segment_size from template loader json was not applied"
 
-# Validate by running vcf2genomicsdb with the generated loader json
-vcf2genomicsdb -r 1 $WORKSPACE/loader.json
+# Fail if same field in INFO and FORMAT have different types
+create_sample_list inconsistent_DP_t0.vcf.gz
+run_command "vcf2genomicsdb_init -w $WORKSPACE -s $SAMPLE_LIST -o" ERR
```

Comment:
I had to remind myself what it was @kgururaj had done with duplicate fields a while back. I'm fine with not duplicating that logic here as well. IMO, we can keep `vcf2genomicsdb_init` as a source of initial config files, and for these rare special cases users can fix up the configurations if need be.

Reply:
Sounds good. Thanks @mlathara